readme.md

```diff
@@ -68,27 +68,15 @@ Additionally, you'll have to do a fraction of the work: one person can work on t
 
```

Comment:
^^ can we kill the line about people completing other work? if they finish early, cool, let them do that. But i don't think we need to be explicit about that in the assignment prompt.  could risk inviting some folks to screw around during the lab and mess up their teammates.